db/db.js

```diff
@@ -379,6 +379,24 @@ async function getLocationData(locationid) {
 
 }
 
+// Retrieves the locationid corresponding to a particle device coreID
+
+async function getLocationFromCoreID(coreID){
+    try{
+        const results = await pool.query("SELECT (locationid) FROM locations WHERE door_coreID = $1 OR radar_coreID = $1", [coreID]);
+        if(results == undefined){
+            console.log('Error: No location with associated coreID exists')
+            return null;
+        }
+        else{
+            return results.rows[0]; 
```

Comment:
This sends you back an object with just the `locationid` key, right? It would probably be nicer for your calling function if you just return the value of `results.rows[0].locationid`.